test/storage/backup/avro/avro-integration.test.ts

```
@@ -0,0 +1,376 @@
+      // Create Avro file with test data
+      const testData = [{ id: '1', created_at: new Date(), label: 'test', metadata: {} }];
+      const backupManager2 = new BackupManager(testConfig, mockStorage);
+      
+      // We'll need to create the Avro file manually or use the serializer
+      // Create proper Avro buffer using AvroSerializer
+      const { AvroSerializer } = await import('../../../../src/storage/backup/avro/avro-serializer');
+      const { AvroSchemaGenerator } = await import('../../../../src/storage/backup/avro/avro-schema-generator');
+      const serializer = new AvroSerializer(new AvroSchemaGenerator());
+      
+      // Create test data that matches snapshots table structure
+      const testData = [{ id: '1', created_at: new Date().toISOString(), label: 'test' }];
```

Biome (2.1.2)

[error] Shouldn't redeclare 'testData'. Consider to delete it or rename it.

'testData' is defined here:


(lint/suspicious/noRedeclare)